### ISSUE TITLE
ggschema TypeScript project

### ISSUE THREAD

**Issue by ijlyttle** [vegawidget/ggvega#17]
Following a virtual discussion with @wenyuyangpku, we plan to make a second TS project, `ggschema`:
 
- in the TS context, ggevga could depend on ggschema 
  - ggschema could (?) build its classes and make them available to ggvega
  - ggvega could (?) use classes to access ggschema elements and other classes to write Vega-Lite elements     

The feasibility of (?) items to be investigated by Wenyu.

```
ggvega/
  src-ext/
    TypeScript/
      ggvega/      
      ggschema/    
```

**Comment by ijlyttle** [comment]
Perhaps this will be the issue that #23 will close.

**Comment by ijlyttle** [comment]
closed in #23
